.eslintrc.json

```
@@ -51,7 +51,7 @@
         "bracketSpacing": true,
         "arrowParens": "always",
         "proseWrap": "preserve",
-        "endOfLine": "lf"
+        "endOfLine": "true"
```

The check code test is failing because of this line and some prettier errors.
```suggestion
        "endOfLine": "lf"
```

---

The Eslint in my environment throws error due to this line before the update and I'm not able to code.